src/stories/datadisplay/AvatarGroup.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { Meta, Story } from "@storybook/react";
-import Avatar from "../../components/Avatar/Avatar";
+import Avatar from "../../components/Avatar";
 import { AvatarGroup, AvatarGroupProps } from "../../components/AvatarGroup";
```

Comment:
`import AvatarGroup, { AvatarGroupProps }`

(at least that's how it's done for other components)